test/relationships.test.ts

```diff
@@ -38,6 +38,42 @@ test('nested query with selective fields', async () => {
             "id": 4,
             "message": "Some message on channel wihtout details",
           },
+          Object {
+            "id": 11,
+            "message": "foo",
+          },
+          Object {
+            "id": 3,
+            "message": "foo",
+          },
+          Object {
+            "id": 12,
+            "message": "test1",
+          },
+          Object {
+            "id": 13,
+            "message": "test1",
+          },
+          Object {
+            "id": 14,
+            "message": "test1",
+          },
+          Object {
+            "id": 15,
+            "message": "updated",
+          },
+          Object {
+            "id": 16,
+            "message": "test3",
+          },
+          Object {
+            "id": 17,
+            "message": "test3",
+          },
+          Object {
+            "id": 18,
+            "message": "test3",
+          },
```

Comment:
Let's delete these rows where they were inserted? One test should affect the others